bridges/snowbridge/pallets/ethereum-client/src/lib.rs

```diff
@@ -191,25 +174,6 @@ pub mod pallet {
 	pub(super) type NextSyncCommittee<T: Config> =
 		StorageValue<_, SyncCommitteePrepared, ValueQuery>;
 
-	/// Latest imported execution header
```

Comment:
A migration for removal those values will be added later, right?

Reply:
Yes, definitely. As soon as Multi-block Migrations are ready on BH.

Reply:
maybe this could be useful: https://github.com/paritytech/polkadot-sdk/pull/3828

Reply:
Looks great! Will that feature be OK even if all the storage items cannot be deleted in one block?